### PR TITLE
Adds readme

### PR DESCRIPTION
**Pull request by ravenbuilder934**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- Adds a short `readme` so potential translators know where to go
-
-

Review please

### CODE REVIEW

**Comment by Nightenom** [comment]
Might be good to also add our pages like discord or minecolonies.com

**Comment by ravenbuilder934** [comment]
Like that?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Structurize
+
+Structurize is a Minecraft mod to aid in designing and placing structures. See its Curseforge page [here](https://www.curseforge.com/minecraft/mc-mods/structurize).
+
+Structurize is made by the same team that makes [MineColonies](https://github.com/ldtteam/minecolonies/issues). You can find our website [here](https://www.minecolonies.com/).
```
Comment (Nightenom) [inline README.md]: Are you sure about the issue link here?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Chat with us and join our amazing community on [Discord](https://www.discord.minecolonies.com)!
+
+### Contributing
+
+There are a few ways you can contribute! 
+
+* Write code for the mod itself (for a tutorial on setting this up, see MineColonies's [readme](https://github.com/ldtteam/minecolonies/blob/version/1.16.3/README.md)).
```
Comment (Nightenom) [inline README.md]:
```suggestion
* Write code for the mod itself (for a tutorial on setting this up, see MineColonies's [readme](https://github.com/ldtteam/minecolonies#readme)).
```
This should be more permanent than blob/1.16.3

File: README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+* Help translate Structurize on [Crowdin](https://crowdin.com/project/structurize).
+* Help with the documentation on our [wiki](https://wiki.minecolonies.ldtteam.com/) ([source](https://github.com/ldtteam/MinecoloniesWiki/issues)).
```
Comment (Nightenom) [inline README.md]: Are you sure about the issue link here? source != issues to me

**Review by Nightenom — DISMISSED** [review]
Yep, nice one, just not sure about the issues link tho

File: README.md (hunk 1 above)
Comment (ravenbuilder934) [reply README.md]: It works ¯\_(ツ)_/¯

Comment (Nightenom) [reply README.md]: I meant it should point to the root of the repo (github.com/ldtteam/minecolonies), not to issues :D

Comment (ravenbuilder934) [reply README.md]: Yeah just realized that, thought you meant the missing `www` :P 
I'll change it

**Review by Nightenom — APPROVED** [review]

**Pull request merged**